### PR TITLE
making layout component

### PR DESCRIPTION
**Pull request by ShrutiSachpara**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,12 +2,20 @@ import { NgModule } from '@angular/core';
...
 import { LoginComponent } from './login/login.component';
 import { ForgotPasswordComponent } from './forgot-password/forgot-password.component';
+import { LayoutComponent } from './layout/layout.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
 
 const routes: Routes = [
   { path: '', redirectTo: 'login', pathMatch: 'full' },
   { path: 'login', component: LoginComponent },
 { path: 'verifyEmail', component: ForgotPasswordComponent },
   //{ path: 'updatePassword', component: ForgotPasswordComponent },
```
Comment (krishna2919) [inline src/app/app-routing.module.ts]: please remove commented code from here

File: src/app/layout/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+              alt="user avatar"
+            />
+            <div class="user-info ps-3">
+              <p class="user-name mb-0">Admin</p>
+              <p class="designattion mb-0">Admin</p>
+            </div>
+          </a>
+          <ul class="dropdown-menu dropdown-menu-end">
+            <li>
+              <a class="dropdown-item" href="javascript:;"
```
Comment (krishna2919) [inline src/app/layout/header/header.component.html]: remove "javascript:;

File: src/app/layout/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+              <a
+                class="nav-link dropdown-toggle dropdown-toggle-nocaret"
+                href="#"
+                role="button"
+                data-bs-toggle="dropdown"
+                aria-expanded="false"
+              >
+              </a>
+              <div class="dropdown-menu dropdown-menu-end">
+                <div class="row row-cols-3 g-3 p-3">
```
Comment (krishna2919) [inline src/app/layout/header/header.component.html]: remove this unused dropdowns and static data

File: src/app/layout/header/header.component.html [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+            <li>
+              <a class="dropdown-item" href="javascript:;"
+                ><i class="bx bx-user"></i><span>Profile</span></a
+              >
+            </li>
+            <li>
+              <a class="dropdown-item" href="javascript:;"
+                ><i class="bx bx-cog"></i><span>Settings</span></a
+              >
+            </li>
```
Comment (krishna2919) [inline src/app/layout/header/header.component.html]: also from here remove javascript and also from all code remove that

File: src/app/layout/sidebar/sidebar.component.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          <h4 class="logo-text">Rocker</h4>
+        </div>
+        <div class="toggle-icon ms-auto">
+          <i class="bx bx-arrow-to-left"></i>
+        </div>
+      </div>
+      <ul class="metismenu" id="menu">
+        <li>
+          <a href="javascript:;" class="has-arrow">
+            <div class="parent-icon"><i class="bx bxs-dashboard"></i></div>
```
Comment (krishna2919) [inline src/app/layout/sidebar/sidebar.component.html]: please use common service for menu items

**Review by krishna2919 — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 6]
```diff
@@ -2,12 +2,19 @@ import { NgModule } from '@angular/core';
...
+import { LayoutComponent } from './layout/layout.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
 
 const routes: Routes = [
   { path: '', redirectTo: 'login', pathMatch: 'full' },
   { path: 'login', component: LoginComponent },
-{ path: 'verifyEmail', component: ForgotPasswordComponent },
-  //{ path: 'updatePassword', component: ForgotPasswordComponent },
+  { path: 'verifyEmail', component: ForgotPasswordComponent },
+  { path: 'forgot-password', component: ForgotPasswordComponent },
```
Comment (SwetaaRampariya) [inline src/app/app-routing.module.ts]: please flow one standard for URL adding.

File: src/app/layout/header/header.component.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    <div class="topbar d-flex align-items-center">
+      <nav class="navbar navbar-expand">
+        <div class="mobile-toggle-menu"><i class="bx bx-menu"></i></div>
+
+        <div class="top-menu ms-auto">
+          <ul class="navbar-nav align-items-center">
+            <li class="nav-item dropdown dropdown-large">
+              <a
+                class="nav-link dropdown-toggle dropdown-toggle-nocaret"
+                href="#"
```
Comment (SwetaaRampariya) [inline src/app/layout/header/header.component.html]: Please remove href="#"

File: src/app/layout/header/header.component.html [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+                aria-expanded="false"
+              >
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="user-box dropdown">
+          <a
+            class="d-flex align-items-center nav-link dropdown-toggle dropdown-toggle-nocaret"
+            href="#"
```
Comment (SwetaaRampariya) [inline src/app/layout/header/header.component.html]: Please remove href="#"

File: src/app/layout/header/header.component.html [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+              src="../../assets/image/avatar-2.png"
+              class="user-img"
+              alt="user avatar"
+            />
+            <div class="user-info ps-3">
+              <p class="user-name mb-0">Admin</p>
+              <p class="designattion mb-0"></p>
+            </div>
+          </a>
+          <ul class="dropdown-menu dropdown-menu-end">
```
Comment (SwetaaRampariya) [inline src/app/layout/header/header.component.html:37]: add for-loop here

**Pull request merged**